### PR TITLE
Unit test for shmem_ctx_get_team

### PR DESCRIPTION
**Pull request by wrrobin**
@davidozog Had this unit test for a different reason, but thought it might be useful anyway. Could you please check if I missed any other condition to check for this routine?

### CODE REVIEW

File: test/unit/shmem_ctx_get_team.c [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        fprintf(stderr, "Error in return values for SHMEM_CTX_DEFAULT\n");
+        shmem_global_exit(2);
+    }
+
+    /* test for a created context on a user defined team*/
+    shmem_team_t         new_team;
+    shmem_team_config_t  *config;
+
+    config = NULL;
+    int npes   = shmem_n_pes();
```
Comment (davidozog) [inline test/unit/shmem_ctx_get_team.c:62]: is this spacing intentional?

File: test/unit/shmem_ctx_get_team.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    ret = shmem_team_split_strided(SHMEM_TEAM_WORLD, 0, 2, (npes + 1) / 2,
+                                   config, 0, &new_team);
+    if (new_team != SHMEM_TEAM_INVALID) {
+        shmem_ctx_t team_ctx;
+        if (!shmem_team_create_ctx(new_team, (long) 0, &team_ctx)) {
```
Comment (davidozog) [inline test/unit/shmem_ctx_get_team.c:68]:
```suggestion
        if (!shmem_team_create_ctx(new_team, 0L, &team_ctx)) {
```
might be preferred

**Review by davidozog — APPROVED** [review]
Looks good, thanks Wasi.

I only provided a couple ridiculously minor suggestions.

File: test/unit/shmem_ctx_get_team.c (hunk 1 above)
Comment (wrrobin) [reply test/unit/shmem_ctx_get_team.c:62]: Nope. Will corect.

**Comment by wrrobin** [comment]
@davidozog Is this good enough to be merged?

**Review by davidozog — APPROVED** [review]
Good to go, thanks @wrrobin!

**Pull request merged**